### PR TITLE
Arnaudmz/icons fix

### PR DESCRIPTION
**Pull request by arnaudmz**
This P/R patch a small hickup in rolebinding icon and update Readme.md with a link to a sample GSlide desk

### CODE REVIEW

File: icons/README.md [hunk 1]
```diff
@@ -74,6 +74,10 @@ There is 2 types of icons
 #### Exposed Pod with 3 replicas
 ![](./docs/k8s-exposed-pod.png)
 
+### Slide Deck
+
+[Kubernetes_Icons_GSlide](https://docs.google.com/presentation/d/15h_MHjR2fzXIiGZniUdHok_FP07u1L8MAX5cN1r0j4U/edit)
```
Comment (cblecker) [inline icons/README.md]: why do we need this?

Comment (arnaudmz) [reply icons/README.md]: This kind of sample was initially included as it a much easier way to start with a sample slide deck instead on importing all PNG files.

After some color change, we had to update it.

Comment (arnaudmz) [reply icons/README.md]: @yastij

Comment (yastij) [reply icons/README.md]: @cblecker 

+100 this is super useful to start with, I've been using it extensively as starter for my slide-deck.

**Pull request merged**